src/geometries/primitives.jl

```diff
@@ -31,3 +31,4 @@ include("primitives/frustum.jl")
 include("primitives/frustumsurface.jl")
 include("primitives/paraboloidsurface.jl")
 include("primitives/torus.jl")
+include("primitives/parametrizedcurve.jl")
```

Comment:
please move this include to after bezier.jl

Reply:
Done in [9822d71](https://github.com/JuliaGeometry/Meshes.jl/pull/1093/commits/9822d71f3cd116dce52f02a96632646a726713de).